src/v3/src/components/PhoneAuthenticator/PhoneAuthenticator.tsx

```diff
@@ -231,81 +176,45 @@ const PhoneAuthenticator: UISchemaElementComponent<UISchemaElementComponentWithV
         flexWrap="wrap"
-        dir="ltr"
+        flexDirection={languageDirection === 'rtl' ? 'row-reverse' : 'row'}
       >
         <Box
           width={showExtension ? 0.7 : 1}
-          marginInlineEnd={showExtension ? 2 : 0}
+          sx={{
+            marginRight: showExtension ? tokens.Spacing2 : tokens.Spacing0,
+          }}
         >
-          <InputLabel
-            htmlFor={fieldName}
-            // To prevent asterisk from shifting far right
-            sx={{ justifyContent: showAsterisk ? 'flex-start' : undefined }}
-            // label should remain in rtl format if rtl language is set
-            dir={languageDirection}
-          >
-            {mainLabel}
-            {showAsterisk && (
-              <Box
-                component="span"
-                sx={{
-                  marginInlineStart: tokens.Spacing2,
-                  marginInlineEnd: tokens.Spacing2,
-                }}
-                className="no-translate"
-                aria-hidden
-              >
-                *
-              </Box>
-            )}
-            {required === false && (
-              <Typography variant="subtitle1">{optionalLabel}</Typography>
-            )}
-          </InputLabel>
-          <InputBase
-            type="tel"
-            name={fieldName}
+          <TextField
+            autoCompleteType={autocomplete}
+            errorMessage={errorMessage}
+            errorMessageList={errorMessageList}
             id={fieldName}
-            error={phoneHasErrors}
-            disabled={loading}
-            dir="ltr"
+            inputMode={inputmode}
+            isDisabled={loading}
+            isFullWidth
+            isOptional={!required}
+            label={mainLabel ?? ''}
+            name={fieldName}
+            onBlur={(e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+              const formattedPhone = formatPhone(e?.currentTarget?.value, phoneCode, extension);
+              handleBlur?.(formattedPhone);
+            }}
             onChange={(e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
               // Set new phone value without phone code
               setPhone(e.currentTarget.value);
               setPhoneChanged(true);
             }}
-            onBlur={(e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
-              const formattedPhone = formatPhone(e?.currentTarget?.value, phoneCode, extension);
-              handleBlur?.(formattedPhone);
-            }}
             startAdornment={(
-              <InputAdornment
+              <Box
                 component="span"
-                position="start"
-                className="no-translate"
-                sx={{
-                  // physical properties OK because parent InputBase component
-                  // is always set to "ltr"
-                  marginRight: tokens.Spacing3,
-                  marginLeft: 0,
-                }}
+                translate="no"
```

Comment:
Odyssey wraps whatever is passed into `startAdornment` in an `InputAdornment` component so we only need to wrap the phone extension text in a `translate="no"`